src/CRTM_LifeCycle.f90

```diff
@@ -673,40 +673,44 @@ FUNCTION CRTM_Init( &
       CALL Display_Message( ROUTINE_NAME, TRIM(msg)//TRIM(pid_msg), err_stat )
       RETURN
     END IF
-
+  
 
     ! Specify sensor-independent coefficient filenames
     ! ...Default File_Path
-    Default_File_Path = ''
+    IF ( PRESENT(File_Path) ) THEN
+            Default_File_Path = File_Path ! Can be overridden later by NC_File_Path
+    ELSE
+            Default_File_Path = ''
+    END IF
     ! ...Default filenames
     Default_Aerosol_Model       = 'CRTM'
-    Default_AerosolCoeff_File   = 'AerosolCoeff.bin'
+    Default_AerosolCoeff_File   = 'AerosolCoeff.nc'
```

Comment:
Haven't tested this branch yet, waiting for fix folder updates on coefficient filenames (nc4 --> nc)

Reply:
@chengdang working on it now -- not ready yet, so ignore pushes to this PR until I've given the thumbs up.